src/HabitTracker.ts

```diff
@@ -48,7 +48,7 @@ function getDaysDifference(startDateId, endDateId) {
 	const end = new Date(endDateId)
 	const oneDay = 24 * 60 * 60 * 1000 // hours * minutes * seconds * milliseconds
 
-	const diffInTime = Math.abs(end - start)
+	const diffInTime = Math.abs(end.getTime() - start.getTime())
```

Comment:
I don't get why this is needed. Both before and after return the same thing
<img width="214" alt="Google Chrome dev tools" src="https://github.com/zoreet/habit-tracker/assets/668132/cc29892e-3a16-4ded-8e9f-bf5c75a5fcd1">

Reply:
The thing is this version is cleaner since it specifies exactly what is happening, instead of relying on a side effect.  At least, that is how I see it when trying to build with typescript.